File: bindings/python/omi_api.py
class OmiApi:
    def __init__(self):
        self.endpoint = None
    def connect_tsdb(self, host, port, proto="udp"):
        self.endpoint = (host, port, proto)
    def stream_event(self, sym, price, size, ts):
        return {"ts": ts, "sym": sym, "price": price, "size": size}
    def correlate_order(self, a_id, b_id):
        return {"a": a_id, "b": b_id}
    def decode_pcap(self, path, script):
        return {"pcap": path, "script": script}
    def configure_time(self, hardware_offset_ns, ptp_error_ns):
        return {"hw": hardware_offset_ns, "ptp": ptp_error_ns}

File: bindings/go/omiapi/omiapi.go
package omiapi
type Endpoint struct{ Host string; Port int; Proto string }
type OmiApi struct{ Endpoint *Endpoint }
func New() *OmiApi { return &OmiApi{} }
func (o *OmiApi) ConnectTSDB(host string, port int, proto string) { o.Endpoint = &Endpoint{Host: host, Port: port, Proto: proto} }
func (o *OmiApi) StreamEvent(sym string, price float64, size int, ts float64) map[string]interface{} {
	return map[string]interface{}{"ts": ts, "sym": sym, "price": price, "size": size}
}
func (o *OmiApi) CorrelateOrder(a string, b string) map[string]string { return map[string]string{"a": a, "b": b} }
func (o *OmiApi) DecodePcap(path string, script string) map[string]string { return map[string]string{"pcap": path, "script": script} }
func (o *OmiApi) ConfigureTime(hardwareOffsetNs int64, ptpErrorNs int64) map[string]int64 {
	return map[string]int64{"hw": hardwareOffsetNs, "ptp": ptpErrorNs}
}
